### PR TITLE
Landing text update

### LINKED ISSUE #258: Add link to OCDS documentation primers in the Registry
Suggestion is to add three links to Registry:

1. [What is the OCDS and why use it?](https://standard.open-contracting.org/latest/en/primer/what/)
2. [How does the OCDS work?](https://standard.open-contracting.org/latest/en/primer/how/)”
3. [How is OCDS data published?](https://standard.open-contracting.org/latest/en/primer/releases_and_records/)

On the homepage we can update the text on the [homepage](https://data.open-contracting.org/en/), with the links 1 and 3 as shown.

----

### **Data by country**
We've designed the Data Registry to help you find and access public procurement datasets that are available in the [OCDS format](https://standard.open-contracting.org/latest/en/primer/what/). 

Currently, you can access data from over 50 publishers from around the world as **compiled releases** with the latest value of each field. [Learn about how OCDS data is published](https://standard.open-contracting.org/latest/en/primer/releases_and_records/).

----

On the Data detials page ([example](https://data.open-contracting.org/en/publication/17)) we can add a sentence at the top of the section "Data available" with the following text: 

"An individual contracting process has several different stages: tendering, awarding, contracting and implementation. You can dive into the details of the contracting stages in [this tutorial](https://standard.open-contracting.org/latest/en/primer/how/)."

### PR DESCRIPTION
**Pull request by yolile**
closes #258

### CODE REVIEW

**Comment by yolile** [comment]
For reference, the pages now look like this:
image
image

I also added all the missing translations.

File: data_registry/locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -137,127 +136,139 @@ msgstr "Language:"
...
-#: data_registry/templates/detail.html:157
+#: data_registry/templates/detail.html:154
+msgid ""
+"An individual contracting process has several different stages: tendering, "
+"awarding, contracting and implementation. You can dive into the details of "
+"the contracting stages in this <a href=\"https://standard.open-"
+"contracting.org/latest/en/primer/how/\" target=\"_blank\">tutorial</a>."
+msgstr ""
+"An individual contracting process has several different stages: tendering, "
+"awarding, contracting and implementation. You can dive into the details of "
```
Comment (jpmckinney) [inline data_registry/locale/en/LC_MESSAGES/django.po:151]: We avoid metaphorical language, to be easier on non-native English speakers.

```suggestion
"awarding, contracting and implementation. You can learn about "
```

File: data_registry/locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -137,127 +136,139 @@ msgstr "Language:"
...
 
-#: data_registry/templates/detail.html:157
+#: data_registry/templates/detail.html:154
+msgid ""
+"An individual contracting process has several different stages: tendering, "
+"awarding, contracting and implementation. You can dive into the details of "
+"the contracting stages in this <a href=\"https://standard.open-"
+"contracting.org/latest/en/primer/how/\" target=\"_blank\">tutorial</a>."
+msgstr ""
+"An individual contracting process has several different stages: tendering, "
```
Comment (jpmckinney) [inline data_registry/locale/en/LC_MESSAGES/django.po:150]:
```suggestion
"A contracting process has several stages: tendering, "
```

File: data_registry/locale/en/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -137,127 +136,139 @@ msgstr "Language:"
...
+"awarding, contracting and implementation. You can dive into the details of "
+"the contracting stages in this <a href=\"https://standard.open-"
+"contracting.org/latest/en/primer/how/\" target=\"_blank\">tutorial</a>."
```
Comment (jpmckinney) [inline data_registry/locale/en/LC_MESSAGES/django.po:153]:
```suggestion
"the contracting stages in <a href=\"https://standard.open-"
"contracting.org/latest/en/primer/how/\" target=\"_blank\">this tutorial</a>."
```

**Comment by jpmckinney** [comment]
Made some edits to `en` (no need to change the template, etc. as the changes are style not content).

**Comment by yolile** [comment]
> Suggested some edits to en (no need to change the template, etc. as the changes are style not content).

Okey, I will update transifex then

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**